ingestion/src/metadata/examples/workflows/alationsink.yaml

```diff
@@ -0,0 +1,34 @@
+source:
+  type: AlationSink
+  serviceName: local_alation_sink
+  serviceConnection:
+    config:
+      type: AlationSink
+      hostPort: https://alation.example.com
+      # Select one authentication type from below
+      # For Basic Authentication
+      authType:
+        username: user_name
+        password: password
+      # # For Access Token Authentication
+      # authType:
+      #   accessToken: access_token
+      projectName: Test
+      paginationLimit: 10
+      # datasourceLinks: {
+      #   "23": "om_service_name.om_db_name",
+      #   "24": "om_service_name_two.om_db_name_two",
+      # }
+  sourceConfig:
+    config:
+        type: DatabaseMetadata
+sink:
+  type: metadata-rest
```

Comment:
why is it metadata-rest?

Reply:
Currently we don't have specific sink for the metadata sources

Reply:
we did not since we always relied to OM as a sink before. Since this is an Alation Sink, maybe we should create the right sink class?